test/fixtures/rego/governance/governance.rego

```diff
@@ -0,0 +1,23 @@
+package governance
+
+local_signer_identities := [
+    {
+        "issuer": "https://kubernetes.default.svc.cluster.local",
+        "subject": "https://kubernetes.io/namespaces/default/serviceaccounts/default",
+    }
+]
+
+ci_signer_identities := [
```

Comment:
We could also use the k8s identity in CI, but IMO it's better to test the flow that actually gets used (ambient OIDC)